workitem/field_definition.go

```diff
@@ -55,20 +55,30 @@ type FieldType interface {
 	ConvertFromModel(value interface{}) (interface{}, error)
 	// Implement the Equaler interface
 	Equal(u convert.Equaler) bool
+	// DefaultValue is called if a non-required field is not specified. In it's
```

Comment:
Why Default if it's non required? Shouldn't it Default in the case where it's non specified but required? Or be flagged if it should default if not specified in general?

Reply:
You're right. I've fixed the comment here: bfc2066ea508efc960c463f3e829430278bb477e. It was a left-over from an earlier idea. In general a default value should be applied if no value is given. It doesn't matter if the field is required or not.